docs/config-file.md

```diff
@@ -436,9 +436,8 @@ settings.
 
 ### `custom_settings.synonyms` _Optional_
 
-`custom_settings` can include a synonyms key that is an array of synonyms. This
-array includes up to 20 elements. Each element is an array of one-word synonyms
-which can be used interchangeably.
+`custom_settings` can include a synonyms key that is an array of synonyms. Each
```

Comment:
```suggestion
`custom_settings` can include a synonym key that is an array of synonyms. Each
```

Reply:
The key must be `synonyms`. I will wrap this word between ` to put the emphasise on this setting.